packages/atomizer/src/blendmodes.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
```

Comment:
I wasn't sure about the naming convention here since we have some dupe rules starting with same letter eg `lighten` & `luminosity` also the order is from the mdn page, I could alphasort...

Reply:
You can follow the convention here https://acss.io/guides/syntax.html#%3Cvalue%3E

Reply:
> If two values share the same initial letter, then the next value in alphabetical order is [abbreviated](http://en.wikipedia.org/wiki/Abbreviation) in [contracted](http://en.wikipedia.org/wiki/Contraction_%28grammar%29) form.

That's not something I am aware of, so would screen be `sc` or more like `scrn`?

Reply:
Since these are values, I think `sc` is short and succinct.